README.md

```diff
@@ -123,15 +153,15 @@ or using home dir expansion:
 const keyPair = await getKeypairFromFile("~/code/solana/demos/steve.json");
 ```
 
-### getKeypairFromEnvironment()
+### getKeypairFromEnvironment(environmentVariable)
 
 Gets a keypair from a secret key stored in an environment variable. This is typically used to load secret keys from [env files](https://stackoverflow.com/questions/68267862/what-is-an-env-or-dotenv-file-exactly).
 
 ```typescript
 const keyPair = await getKeypairFromEnvironment("SECRET_KEY");
 ```
 
-### addKeypairToEnvFile()
+### addKeypairToEnvFile(keypair, environmentVariable, [file])
```

Comment:
the `addKeypairToEnvFile` takes a string string as that last arg, not an array

Reply:
Thanks again.